app/models/export/activity_forecast_columns.rb

```diff
@@ -57,10 +62,12 @@ def forecast_financial_quarter_range
     @_forecast_quarter_range ||= begin
       return [] if all_financial_quarters_with_forecasts.blank?
 
-      if @report.nil?
-        Range.new(all_financial_quarters_with_forecasts.min, all_financial_quarters_with_forecasts.max)
-      else
+      if @starting_financial_quarter.present?
+        Range.new(@starting_financial_quarter, all_financial_quarters_with_forecasts.max)
+      elsif @report.present?
```

Comment:
would it be helpful to wrap these in more descriptive method names -- so that it's clear why a caller would use one or other param? e.g.

```rb
def exporting_most_recent_forecasts?
  @starting_financial_quarter.present?
end

def exporting_forecasts_for_given_report?
  @report.present?
end
```

Reply:
Yes! Thank you!